__tests__/app.test.js

```diff
@@ -1,3 +1,4 @@
+const { TestWatcher } = require("jest");
```

Comment:
I don't know what this is, but I'd assume it's sneaked in as an auto-import. Please remove!